### PR TITLE
Add eslint config

### PR DESCRIPTION
**Pull request by mweibel**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  "extends": [
+    "react-app",
+    "plugin:react/recommended",
+    "prettier",
+    "prettier/react"
+  ],
+  "plugins": ["react", "prettier", "import"],
+  "rules": {
+    "prettier/prettier": [2, { "singleQuote": true }],
+    "import/no-extraneous-dependencies": 0,
```
Comment (Hanse) [inline .eslintrc]: Can you remove the rules we don't need? I.e. remove all and re-add them if an error occurs. Most of them are only necessary for the project I linked to in #18. `prettier/prettier` should at least stay, but I believe the others aren't necessary.

Comment (mweibel) [reply .eslintrc]: 👍 Changed it, did you mean it like I did it now?

**Comment by Hanse** [comment]
Can you remove the pakage-lock.json? The project prefers yarn

**Comment by mweibel** [comment]
Fixed & rebased.

**Comment by Hanse** [comment]
Thanks!

**Comment by mweibel** [comment]
Thanks for merging! Will you publish a new version soon?

**Comment by Hanse** [comment]
Yep, released 1.3.0 just now 🎉

**Comment by mweibel** [comment]
awesome 👍

**Pull request merged**